_posts/2018-04-21-welcome-to-this-week-in-rust.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Welcome to this week in Rust-WASM! 001"
+date:   2018-04-21 21:15:00 +0200
+---
+Hello and welcome to the another issue of *This Week in Rust+WASM*!
+
+[Rust](http://rust-lang.org) is a systems language pursuing the trifecta: safety, concurrency, and speed.
+
+[WASM](http://webassembly.org) is designed as a portable target for compilation of high-level languages like C/C++/Rust, enabling deployment on the web for client and server applications.
+
+This is a weekly summary of its progress and community.
+
+Tweet us at [@rustwasm](https://twitter.com/rustwasm) or send us a pull request. Want to get involved? [We love contributions](https://github.com/rust-lang-nursery/rust-wasm/blob/master/README.md)
+
+## News
+* [WebAssembly Studio - Online IDE](https://hacks.mozilla.org/2018/04/sneak-peek-at-webassembly-studio/)
+* [Game of Life - Build your game with WebAssembly](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/introduction.html)
+* [Svelte -> Twiggy (A code size profiler for WASM written in Rust)](https://github.com/rustwasm/twiggy)
+    - [Analyze bloat from monomorphisation of generic functions](https://github.com/rustwasm/twiggy/pull/40)
+* [GGez on Rust-WASM](https://wiki.alopex.li/GgezOnWasm)
```

Comment:
Suggest: "What will it take to get `ggez` (a lightweight 2D games framework) on WebAssembly?"